scout/models/omics_variant.py

```diff
@@ -231,14 +231,6 @@ def set_omics_variant_id(cls, values) -> "OmicsVariantLoader":
         )
         return values
 
-    @model_validator(mode="before")
-    def set_sample_display_name(cls, values) -> "OmicsVariantLoader":
-        """Set a display name."""
-        values["display_name"] = values.get(
-            "display_name", values.get("sample_name", values.get("individual_id"))
```

Comment:
display_name is the display_name of the variant, no? I don't think this check is needed

Reply:
Right, but there is no display name in the tsv file, and it's a mandatory field on variants. This was a placeholder. Let's fill it with something else; perhaps echo the variant_id, or make a light version of that more prominently focusing on coords or the variant qualification.

Reply:
There's this other check that sets the display_name of the variant --> https://github.com/Clinical-Genomics/scout/blob/0611bcec0b1e8d18f0e939c9c77643412ed861fc/scout/models/omics_variant.py#L199

Reply:
😂 Good, that would be what I intended. I agree, let's remove the placeholder then! 😁